conf/defaults.ini

```diff
@@ -279,11 +279,17 @@ editors_can_admin = false
 # Login cookie name
 login_cookie_name = grafana_session
 
-# The lifetime (days) an authenticated user can be inactive before being required to login at next visit. Default is 7 days.
-login_maximum_inactive_lifetime_days = 7
+# Deprecated, use login_maximum_inactive_lifetime_duration instead
+login_maximum_inactive_lifetime_days = 
```

Comment:
We don't want to keep deprecated references in defaults.ini. To handle deprecation notice we usually add a [version upgrade note](https://grafana.com/docs/grafana/latest/installation/upgrading/) and/or add it to the changelog.

```suggestion
```

Reply:
I'm having a problem getting the tests to work as the value passed for login_maximum_inactive_lifetime_days does not seem to "reach" the config when I removed the entry in defaults.ini. Putting it back made it work again, am I missing a change somewhere?

Reply:
I suggest you use a test similar to this: https://github.com/grafana/grafana/blob/master/pkg/setting/setting_plugins_test.go

Difference are that you populate ini file "in memory" in the setup of test. Then I would also suggest that you call the `readAuthSettings` function in the test to verify instead of using `cfg.Load(...`.

Let me know if you need anything else